lib/plaid/models.rb

```diff
@@ -233,6 +233,44 @@ class NumberEFT < BaseModel
       property :institution
     end
 
+    # Public: A representation of an international account number.
```

Comment:
We can add examples for `NumberInternational` and `NumberBACS` fields similar to examples for `NumberACH`. Can copy these examples from draft documentation for customers.

Reply:
lets do it as a follow up (i'd want to use the same examples that we are going to publish on the docs).